creator-node/src/routes/tracks.js

```diff
@@ -381,7 +380,8 @@ module.exports = function (app) {
           transaction
         })
 
-        if (trackFiles.length !== trackSegmentCIDs.length) {
+        if (trackFiles.length < trackSegmentCIDs.length) {
```

Comment:
this also takes care of the case where a previous track_content left unassociated files. we'd error if the number of files is not exactly equal (so duplicates would break). this way we'll associate it all under this trackId. See https://github.com/AudiusProject/audius-protocol/blame/3bf6a8137ae25529583f21ac259562c712e8b3b5/creator-node/src/routes/tracks.js for previous behavior